dockerfiles/pangenie/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# PanGenie docker file.
+# In this Dockerfile PanGenie is installed via conda (recommended approach
+# by its maintainers https://bitbucket.org/jana_ebler/pangenie/src/master/).
+#
+# A note on the base image, miniconda3:
+# Installation via conda requires activation of PanGenie conda environmet;
+# however, that requires initialization bash to use conda (error message:
+# "Your shell has not been properly configured to use 'conda activate'".).
+# Conda initialization and bash profile reload via the following does not
+# resolve the issue.
+#   RUN bash miniconda.sh -b -p /opt/conda && \
+#       /opt/conda/bin/conda init bash && \
+#       bash -c "source ${HOME}/.bashrc"
+#
+# A similar issue is detailed in the following blog post; therefore, as
+# suggested in this blog post, this docker files is using
+# 'continuumio/miniconda3' as base image.
+#   https://pythonspeed.com/articles/activate-conda-dockerfile/
+
+FROM continuumio/miniconda3
```

Comment:
Can you peg this to a specific tag?

Reply:
Good point, thanks! I pinned the docker image and all the tools installed to their current latest tag and versions respectively.